index.html

```diff
@@ -11,6 +12,121 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+
+  <div class="container">
+
+    <header>
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+      <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+    </header>
+
+    <section class="section1"> 
+      <div class = "section1left">
+        <div class = "section1leftText">
+          <h1>
+            Innovation
+          </h1>
+          <h1>
+            On
+          </h1>
+          <h1>
+            Demand
+          </h1>
+        </div>
+        <button>
+            Get Started
+        </button>
+      </div>
+      <div class = "section1right">
+          <img src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </section>  
+
+    <section class = "section2">
+      <div class ="sectionContent">
+        <h2>
+          Features
+        </h2>
+        <p>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>
+      </div>  
+
+      <div class ="sectionContent">
+        <h2>
+          About
+        </h2>
+        <p>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </p>
+      </div>
+    </section>
+
+    <section class="midImg">
```

Comment:
```suggestion

```
If you just have an image and nothing else in a container, make it a figure rather than a section. However I would like to suggest another way to handle your image altogether, which would involve it not being inside a container at all